### PR TITLE
chore: change calendso strings to cal.com

### PR DESCRIPTION
**Pull request by conrmahr**
Helping with the rebrand to Cal.com!

- change all instances of `Calendso` strings to `Cal.com` excluding namespaces

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -244,7 +244,7 @@ Contributions are what make the open source community such an amazing place to b
 6. In the third page (Test Users), add the Google account(s) you'll using. Make sure the details are correct on the last page of the wizard and your consent screen will be configured.
 7. Now select [Credentials](https://console.cloud.google.com/apis/credentials) from the side pane and then select Create Credentials. Select the OAuth Client ID option.
 8. Select Web Application as the Application Type.
-9. Under Authorized redirect URI's, select Add URI and then add the URI `<CALENDSO URL>/api/integrations/googlecalendar/callback` replacing CALENDSO URL with the URI at which your application runs.
+9. Under Authorized redirect URI's, select Add URI and then add the URI `<CALENDSO URL>/api/integrations/googlecalendar/callback` replacing Cal.com URL with the URI at which your application runs.
```
Comment (baileypumfleet) [inline README.md:247]: <CALENDSO URL> hasn't been changed :)

File: README.md [hunk 2]
```diff
@@ -253,7 +253,7 @@ Contributions are what make the open source community such an amazing place to b
 1. Open [Azure App Registration](https://portal.azure.com/#blade/Microsoft_AAD_IAM/ActiveDirectoryMenuBlade/RegisteredApps) and select New registration
 2. Name your application
 3. Set **Who can use this application or access this API?** to **Accounts in any organizational directory (Any Azure AD directory - Multitenant)**
-4. Set the **Web** redirect URI to `<CALENDSO URL>/api/integrations/office365calendar/callback` replacing CALENDSO URL with the URI at which your application runs.
+4. Set the **Web** redirect URI to `<CALENDSO URL>/api/integrations/office365calendar/callback` replacing Cal.com URL with the URI at which your application runs.
```
Comment (baileypumfleet) [inline README.md:256]: `<CALENDSO URL>` hasn't been changed :)

File: README.md [hunk 3]
```diff
@@ -267,7 +267,7 @@ Contributions are what make the open source community such an amazing place to b
 6. De-select the option to publish the app on the Zoom App Marketplace.
 7. Click "Create".
 8. Now copy the Client ID and Client Secret to your .env file into the `ZOOM_CLIENT_ID` and `ZOOM_CLIENT_SECRET` fields.
-9. Set the Redirect URL for OAuth `<CALENDSO URL>/api/integrations/zoomvideo/callback` replacing CALENDSO URL with the URI at which your application runs.
+9. Set the Redirect URL for OAuth `<CALENDSO URL>/api/integrations/zoomvideo/callback` replacing Cal.com URL with the URI at which your application runs.
```
Comment (baileypumfleet) [inline README.md:270]: `<CALENDSO URL>` hasn't been changed :)

**Review by pumfleet — CHANGES_REQUESTED** [review]
Thank you very much for this amazing contribution! There are a couple of things that need to be replaced, but we really appreciate your PR regardless 😄

**Review by pumfleet — APPROVED** [review]

**Review by PeerRich — APPROVED** [review]
lgtm. surprised why some lines were formatted differently

**Pull request merged**